app/components/container/containerController.js

```diff
@@ -117,22 +117,28 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Ima
   };
 
   $scope.confirmRemove = function () {
+    var title = 'You are about to remove a container.';
     if ($scope.container.State.Running) {
-      ModalService.confirmDeletion(
-        'You are about to remove a running container.',
-        function (confirmed) {
-          if(!confirmed) { return; }
-          $scope.remove();
+      title = 'You are about to remove a running container.';
+    }
+    var text = '<label for="clean" class="control-label text-left">Automatically clean associated volumes</label><label class="switch" style="margin-left: 21px;"><input type="checkbox"><i></i></label>';
```

Comment:
Regarding the css update, I'm talking about removing the `switch` here and try to replace the checkbox inside a bootbox using the existing switch css rules.